devstart.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash
-cd /react-spark/example;
+pushd $SOURCE_DIR/react-spark;
```

Comment:
this block cannot run for personal loans as it happens here: https://github.com/qloan/dev-env/blob/feat/redesign-phase1/docker/includes/cmd.sh#L25